src/index.css

```diff
@@ -14,7 +14,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  height: 100vh;
+  height: 100%;
   background-color: #f3f4f8;
 }
```

Comment:
- [ ] Please stick to the design as much as possible. Please use the same color for buttons and titles. See the image below for reference.
### Your design
![ss](https://user-images.githubusercontent.com/81727988/216776989-d1869817-f622-4265-8fc7-09f39cdbd2ee.png)


###Original deisgn
![jj](https://user-images.githubusercontent.com/81727988/216776995-506e5ebf-8637-497e-931b-6426f1f2f528.png)